### PR TITLE
Add information about ActivityStarter special support for Parceler

### PR DESCRIPTION
**Pull request by MarcinMoskala**

### CODE REVIEW

File: index.adoc [hunk 1]
```diff
@@ -84,6 +84,7 @@ This wrapping and unwrapping technique plays well with the Intent Factory patter
 In addition, Parceler is supported by the following libraries:
 
  * http://androidtransfuse.org/documentation.html#parcel[Transfuse] - Allows `@Parcel` annotated beans to be used with the `@Extra` injection.
+ * https://github.com/MarcinMoskala/ActivityStarter/wiki/Parceler-Arg-Converter-usage[ActivityStarter] - Uses ParcelarArgConverter to allow to use Parceler objects as an arguments passed to Activities (Fragments, Services etc.).
```
Comment (johncarl81) [inline index.adoc:87]: Make `ParcelarArgConverter` code formatted (use ``` character).

**Comment by johncarl81** [comment]
Looking at your implmentation, would you consider changing `ParcelarArgConverter` to `ParcelerArgConverter`?

**Comment by johncarl81** [comment]
Also, it looks like you are using a marker interface to determine if a type is `@Parcel`able or not, why not use the annotation?

File: index.adoc (hunk 1 above)
Comment (johncarl81) [reply index.adoc:87]: Also, move this down to the bottom of the list.

**Comment by MarcinMoskala** [comment]
Please, hold it. I will add native support for Parceler

**Pull request merged**